<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="Admin-PC" name="com.uno.test.sd.SD_SDActivityTest" tests="1" failures="1" timestamp="23 Nov 2018 04:56:32 GMT" time="11.096" errors="0">
  <testcase name="checkFromAndToDateAccordingSelectedDateRange" time="11.096" classname="com.uno.test.sd.SD_SDActivityTest">
    <failure type="java.lang.AssertionError" message="Log Date And Time is not same expected [2018-11-23] but found [00:00:00]">
      <![CDATA[java.lang.AssertionError: Log Date And Time is not same expected [2018-11-23] but found [00:00:00]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at com.uno.pages.sd.SD_EngineerActivityPage.checkFromAndToDateAccordingSelectedDateRange(SD_EngineerActivityPage.java:320)
at com.uno.pages.sd.SD_SDActivityPage.checkFromAndToDateAccordingSelectedDateRange(SD_SDActivityPage.java:557)
at com.uno.test.sd.SD_SDActivityTest.checkFromAndToDateAccordingSelectedDateRange(SD_SDActivityTest.java:76)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkFromAndToDateAccordingSelectedDateRange -->
</testsuite> <!-- com.uno.test.sd.SD_SDActivityTest -->
